editor/import/resource_importer_obj.cpp

```diff
@@ -217,7 +217,8 @@ static Error _parse_obj(const String &p_path, List<Ref<Mesh>> &r_meshes, bool p_
 	Vector<Vector3> vertices;
 	Vector<Vector3> normals;
 	Vector<Vector2> uvs;
-	String name;
+	const String default_name = "Mesh";
+	String name = default_name;
```

Comment:
I would use the class name of the mesh.

Reply:
The default name will be used for multiple classes throughout the import process (`ArrayMesh`, `ImporterMesh` and `ImporterMeshInstance3D`), so I'm not sure if adding significant complexity to use the name of each individual class if no name is set is a good idea.